app/static/js/createEvents.js

```diff
@@ -37,7 +37,9 @@ function format24to12HourTime(timeStr){
  * Run when the webpage is ready for javascript
  */
 $(document).ready(function() {
-    $("#attachmentObject").fileinput()
+    $("#attachmentObject").fileinput({
+        allowedFileExtensions:["pdf","jpg","png","gif", "csv", "docx"]
```

Comment:
Looks good only snag is we want to add `jpeg` just another common image file type.